Tools/wasm/README.md

```diff
@@ -23,16 +23,25 @@ to a repository checkout.
 Christian Heimes maintains a container image with Emscripten SDK, Python
 build dependencies, WASI-SDK, wasmtime, and several additional tools.
 
+From within your local CPython repo clone, run one of the following commands:
+
 ```
 # Fedora, RHEL, CentOS
-podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z quay.io/tiran/cpythonbuild:emsdk3
+podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z -w /python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
 
 # other
-docker run --rm -ti -v $(pwd):/python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
+docker run --rm -ti -v $(pwd):/python-wasm/cpython -w /python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
 ```
 
+Note that when you exit the container, all work outside the mounted cpython
+directory will be lost.
+See the `podman` or `docker` docs for how to preserve the container environment
+across runs.
+
```

Comment:
I suggest that you remove the paragraph. It is confusing and distracting. All instructions in the README operate on data in the mounted directory.

Reply:
```suggestion
```